core-concepts/dynamic-ui/application-layout.md

```diff
@@ -0,0 +1,20 @@
+---
+description: The Application layout can be selected to match the application size to the device it is intended for.
```

Comment:
This section should be used to introduce a user to the concept rather than specify the exact working of the feature. Something like
"Appsmith supports a variety of screen sizes to build UI"